jobs/generation/MetaGenerator.groovy

```diff
@@ -227,11 +227,13 @@ repos.each { repoInfo ->
         return;
     }
 
-    // Make the folders
+    // Make the folders. Save the root folder for overview generation
     def generatorFolder = ''
+    def projectFolder = ''
     for (folderElement in repoInfo.folders) {
         if (generatorFolder == '') {
             generatorFolder = folderElement
+            projectFolder = folderElement
```

Comment:
Yeah, the intention is to give an overview of all full builds for all branches, so I don't have to drill down into each branch to find the status of branches I need to monitor.